src/main/cocoa/CocoaWindow.mm

```diff
@@ -112,13 +112,12 @@
                 CocoaCairoView *view = [[CocoaCairoView alloc] initWithFrame:frame];
                 pCocoaView = view;
                 [pCocoaWindow setContentView:pCocoaView];
+                [pCocoaView startRedrawLoop];
```

Comment:
I think the redraw loop should be started when window becomes visible and stopped when window becomes invisible.